notebooks/College_Demo.ipynb

```diff
@@ -21,7 +21,10 @@
     "\n",
     "data = pd.read_csv('https://raw.githubusercontent.com/lux-org/lux-datasets/master/data/college.csv')\n",
     "engine = create_engine(\"postgresql://testuser:testpass@localhost:5432/testdb\")\n",
-    "data.to_sql(name='college', con=engine, if_exists = 'replace', index=False)"
```

Comment:
Can we place the data upload scripts into a separate file to be imported so that it is not confused with the Lux SQL statements?